app/src/main/java/com/islaharper/dawnofandroid/util/Resource.kt

```diff
@@ -0,0 +1,9 @@
+package com.islaharper.dawnofandroid.util
+
+// Handles ApiResponse
+sealed class Resource<out Any> {
+    object Idle : Resource<Nothing>()
```

Comment:
Example of when Idle is used:
`    private val _apiResponse: MutableState<Resource<ApiResponse>> = mutableStateOf(Resource.Idle)
    val apiResponse: State<Resource<ApiResponse>> = _apiResponse`

It's an addition to the Resource wrapper and is used as an initial state.

https://www.udemy.com/course/secure-user-authentication-android-ktor-server-mongodb/learn/lecture/31216110#overview